src/app/wallet/shared/util/utils.ts

```diff
@@ -166,3 +166,26 @@ export class AddressHelper {
       '' : this.getAddress(event.clipboardData.getData('text'));
   }
 }
+
+export class DateFormatter {
```

Comment:
Could you please use moment js for manage date related all functionality?

Reply:
yes we can I guess :)

Reply:
yup we can I guess